.github/workflows/dispatcher.yml

```diff
@@ -7,7 +7,7 @@ jobs:
   dispatch:
     runs-on: ubuntu-latest
     steps:
-      - name: Slash Command Dispatcher
+      - name: Slash Command Dispatcher Update Refdata
```

Comment:
Update regression data

Reply:
Fixed in https://github.com/tardis-sn/tardis/pull/2423/commits/69b2054d54adc70bab49cc596cf3b316ebb3c52d